addons/account/static/src/views/upload_file_from_data_hook.js

```diff
@@ -0,0 +1,68 @@
+import { rpc } from "@web/core/network/rpc";
+
+
+// Supported file types we need extract on paste
+const supportedFileTypes = ["text/xml", "application/pdf"];
+
+/**
+ * Return function to extract and upload from given dataTransfer.
+ *
+ * @param {dataTransfer} dataTransfer containing text or files.
+ */
+export function uploadFileFromData(dataTransfer) {
+    return async (dataTransfer) => {
+        function isValidUrl(text) {
+            try {
+                const { protocol } = new URL(text);
+                if (protocol === 'https:') {
+                    return true;
+                }
+                console.warn("Not a secure url.");
+            } catch {
+                console.warn("Invalid url.");
+            }
+        }
+
+        function uploadFiles(dataTransfer) {
+            const invalidFiles = [...dataTransfer.items].filter(
+                (item) => item.kind !== "file" || !supportedFileTypes.includes(item.type)
+            );
+            if (invalidFiles.length !== 0) {
+                // don't upload any files if one of them is non supported file type
+                console.warn("Invalid files to extract details.");
+                return;
+            }
+            const selector = '.document_file_uploader.o_input_file';
+            let uploadInput = document.querySelector(selector);
```

Comment:
```suggestion
            let uploadInput = document.querySelector('.document_file_uploader.o_input_file');
```
Don't think we need the variable.